api/WindowClient.json

```
@@ -180,7 +180,7 @@
               "version_added": "36"
             },
             "safari": {
-              "version_added": "11.1"
+              "version_added": false
             },
             "safari_ios": {
               "version_added": "11.3"
```

I suspect this is also true of Safari for iOS, so we should apply similar changes here:

```suggestion
              "version_added": "11.3",
              "partial_implementation": true,
              "notes": "This method exists, but always throws <code>NotSupportedError</code>."
```